lib/resources/Sources.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+var StripeResource = require('../StripeResource');
+var stripeMethod = StripeResource.method;
+var utils = require('../utils');
```

Comment:
I know that imports are a bit of a mess right now, but maybe just for strict correctness, drop both `stripeMethod` and `utils` here; neither are needed.